curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.portuguese.md

```diff
@@ -7,17 +7,19 @@ localeTitle: Use a propriedade flex-direction para criar uma linha
 ---
 
 ## Description
-<section id="description"> Adicionando <code>display: flex</code> para um elemento o transforma em um contêiner flexível. Isso torna possível alinhar quaisquer filhos desse elemento em linhas ou colunas. Você faz isso adicionando a propriedade <code>flex-direction</code> ao item pai e configurando-o para linha ou coluna. A criação de uma linha alinhará os filhos horizontalmente e a criação de uma coluna alinhará os filhos verticalmente. Outras opções para <code>flex-direction</code> são reversão de linha e reversão de coluna. <strong>Nota</strong> <br> O valor padrão da propriedade <code>flex-direction</code> é row. </section>
+<section id="description"> Adicionando <code>display: flex</code> a um elemento o transforma em um contêiner flexível. Isso torna possível alinhar quaisquer filhos desse elemento em linhas ou colunas. Você faz isso adicionando a propriedade <code>flex-direction</code> ao item pai e configurando-a para <code>row</code> ou <code>column</code>. A criação de uma linha alinhará os filhos horizontalmente e a criação de uma coluna alinhará os filhos verticalmente.
+Outras opções para <code>flex-direction</code> são  <code>row-reverse</code> (linha reversa) e <code>column-reverse</code> (coluna reversa).
```

Comment:
Please try to follow the same formatting of the previous translation. Don't break the line when there's no line break previously.